quasar/main.tf

```diff
@@ -519,6 +519,7 @@ resource "aws_db_instance" "quasar-qa" {
   engine                          = "postgres"
   engine_version                  = "11.1"
   instance_class                  = "db.m5.2xlarge"
+  allow_major_version_upgrade     = true
   name                            = "quasar"
   username                        = "${data.aws_ssm_parameter.qa_username.value}"
   password                        = "${data.aws_ssm_parameter.qa_password.value}"
```

Comment:
I think the `parameter_group_name` here needs to be `"${aws_db_parameter_group.quasar-qa-pg11.id}"`.